### PR TITLE
Fixed add profile property

### PR DESCRIPTION
**Pull request by SPSeanLong**
Adding a new profile property was going through the UpdateProfile method instead of AddProfile 
 method. Now uses the AddProfile method when there was no pre-existing id as already used earlier in the same function for same destinction

### CODE REVIEW

File: Oqtane.Client/Modules/Admin/Profiles/Edit.razor [hunk 1]
```diff
@@ -154,8 +154,14 @@
             profile.DefaultValue = _defaultvalue;
             profile.IsRequired = (_isrequired == null ? false : Boolean.Parse(_isrequired));
             profile.IsPrivate = (_isprivate == null ? false : Boolean.Parse(_isprivate));
-            profile = await ProfileService.UpdateProfileAsync(profile);
-            
+            if (_profileid != -1)
+            {
+                profile = await ProfileService.UpdateProfileAsync(profile);
+            }else
```
Comment (hishamco) [inline Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: Format

Comment (hishamco) [reply Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: we can use

```csharp
profile = _profileid != -1
    ? await ProfileService.UpdateProfileAsync(profile)
    : await ProfileService.AddProfileAsync(profile);
```

Comment (sbwalker) [reply Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: My preference is to use the traditional conditional logic as it is much easier for developers to read and understand - which makes the code easier to maintain

Comment (sbwalker) [reply Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: the else should be on its line to be consistent with the rest of the framework code, but I will merge this PR anyways.

Comment (SPSeanLong) [reply Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: To be sure I comply correctly in the future, the desire is

`}

else

{`

Comment (sbwalker) [reply Oqtane.Client/Modules/Admin/Profiles/Edit.razor:160]: Correct ( without the extra line breaks before and after the “else” of course ). If you look at the other framework code you will are the standard is:

If ()
{
...
}
else
{
....
}

**Pull request merged**